src/components/content/conditions/patient-conditions.tsx

```diff
@@ -21,6 +23,12 @@ const PatientConditionsComponent = ({ className, readOnly = false }: PatientCond
   const showAddConditionForm = useAddConditionForm();
   const { t } = useBaseTranslations();
 
+  useEffect(() => {
+    if (!query.isLoading) {
+      Telemetry.reportZAPRecordCount("builder_conditions", query.data?.length);
+    }
+  }, [query.isLoading, query.data]);
```

Comment:
We probably don't want to call `reportZAPRecordCount` unless `query.data` exists
```suggestion
  useEffect(() => {
    if (!query.isLoading && query.data) {
      Telemetry.reportZAPRecordCount("builder_conditions", query.data.length);
    }
  }, [query.isLoading, query.data]);
```